profile/README.md

```diff
@@ -0,0 +1,93 @@
+          <a href="https://github.com/passageidentity/passage-flex-rust">Rust</a><br />
+          <b>Integration Guides:</b>
+          <a href="https://docs.passage.id/flex/aws-cognito" target="_new">Amazon Cognito</a>
+        </sub><br /><br />
+    </td>
+  </tr>
+  <tr>
+    <td valign="top">
+        <h3><img src="https://storage.googleapis.com/passage-docs/github-md-assets/passage-passkey-complete-icon.png"> Passkey <b>Complete</b></h3>
+        <a href="https://docs.passage.id/complete" target="_new">Use Passage as a standalone auth solution →</a>
+        <p>Replace your existing authentication flow or build from scratch with a robust solution for passwordless authentication and customer identity management. In addition to passkeys, you can add other authentication methods including social sign-in, Magic Links, and one-time codes.</p>
+      <sub>Passkey Complete Libraries & SDKs</sub><br />
+      <sub>
+        <b>Frontend:</b>
+        <a href="https://docs.passage.id/complete/elements" target="_new">Passage Elements</a> |
+        <a href="https://docs.passage.id/complete/examples-by-framework/angular" target="_new">Angular</a> |
+        <a href="https://docs.passage.id/complete/examples-by-framework/next-js" target="_new">Next.js</a> |
+        <a href="https://www.npmjs.com/package/@passageidentity/passage-react" target="_new">React</a> |
+        <a href="https://docs.passage.id/complete/examples-by-framework/vue" target="_new">Vue.js</a><br />
+        <b>Native:</b>
+        <a href="https://github.com/passageidentity/passage-android">Android</a> |
+        <a href="https://github.com/passageidentity/passage-swift">Swift</a> |
+        <a href="https://github.com/passageidentity/passage-react-native">React Native</a> |
+        <a href="https://github.com/passageidentity/passage-flutter">Flutter</a><br />
+        <b>Backend:</b>
+        <a href="https://github.com/passageidentity/passage-go">Go</a> |
+        <a href="https://github.com/passageidentity/passage-node">Node.js</a> |
+        <a href="https://github.com/passageidentity/passage-php">PHP</a> |
+        <a href="https://github.com/passageidentity/passage-python">Python</a> |
+        <a href="https://github.com/passageidentity/passage-ruby">Ruby</a>
+      </sub><br /><br />
+    </td>
+  </tr>
+
+  <tr>
+    <td>
+        <h3><img src="https://storage.googleapis.com/passage-docs/github-md-assets/passage-passkey-ready-icon.png"> Passkey <b>Ready</b></h3>
+        <a href="https://docs.passage.id/ready" target="_new">Determine if your users are ready for passkeys →</a>
+        <p>Add a few lines of code to your app or website and Passkey Ready will generate data to determine if your end users are ready for passkeys.</p>
+      <sub>Passkey Ready Libraries & SDKs</sub><br />
+      <sub>
+        <b>Clients:</b>
+          <a href="https://docs.passage.id/passkey-ready/integration" target="_new">JavaScript</a> |
```

Comment:
Just a thought - maybe Web instead of JavaScript since we have android/iOS vs their languages? Take it or leave it!

```suggestion
          <a href="https://docs.passage.id/passkey-ready/integration" target="_new">Web</a> |
```